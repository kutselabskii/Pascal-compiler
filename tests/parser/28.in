program abc;

var a, i: integer;

begin
write('hello' + ' ' + 'world!');
end.