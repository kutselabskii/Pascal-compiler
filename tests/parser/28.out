|__ program
    |-- abc //Program_heading
    |__ //Block
        |-- //Declaration part
        |   |-- var
        |   |   |-- a
        |   |   |   |__  : integer
        |   |   |__ i
        |   |       |__  : integer
        |__ //Statement part
            |__ //Statement sequence
                |-- write
                |   |__ hello world!
